std/algorithm/sorting.d

```diff
@@ -872,31 +876,53 @@ template multiSort(less...) //if (less.length > 1)
             alias ss = SwapStrategy.unstable;
             alias funs = less;
         }
-        alias lessFun = binaryFun!(funs[0]);
 
-        static if (funs.length > 1)
+        static if (funs.length == 1)
+            return sort!fun[0](r);
```

Comment:
I wouldn't move the conversion of the predicate to the instantiation of sort w/o a good reason. So please keep the lessFun alias and we're sure to not trigger any weird instantiation error w/ nested templates or run into semantic order issues.